themes/mCommerceTheme/styles/components/flProductItem.json

```diff
@@ -210,7 +209,7 @@
           "top": 10
         }
       },
-      "paddingTop": 2,
+      "paddingTop": 5,
       "paddingBottom": 2
     },
     "height": 270,
```

Comment:
activityIndicator not centered inside add to basket button

Reply:
DONE